app/controllers/sdg_management/local_targets_controller.rb

```diff
@@ -0,0 +1,50 @@
+class SDGManagement::LocalTargetsController < SDGManagement::BaseController
+  include Translatable
+
+  LocalTarget = ::SDG::LocalTarget
+
+  def index
+    @local_targets = LocalTarget.all
```

Comment:
I think we'd have to use `order` or `sort` to make sure they appear in the order we expect.

Reply:
I added here the Comparable module and using it in a very similar way we do on targets.